.vscode/launch.json

```diff
@@ -11,11 +11,14 @@
                 "--extensionDevelopmentPath=${workspaceRoot}/src/vscode-bicep"
             ],
             "env": {
-                "BICEP_TRACING_ENABLED": "true"
+                "BICEP_TRACING_ENABLED": "true",
+                "DEBUGTELEMETRY": "", // "" or "0" or "false": send telemetry; "1": no telemetry; "verbose": telemetry in console, don't send
```

Comment:
>DEBUGTELEMETRY [](http://example.com/codeflow?start=17&length=14)

Where is this being consumed? (I don't see any references to it.) Is this a standard vscode env var?

Reply:
It's from the vscode-azureextensionui extension.  All of our telemetry is handled by that extension.

Reply:
Added comment

Reply:
Got it. Makes ense.